content/400-reference/200-api-reference/100-prisma-schema-reference.mdx

```diff
@@ -104,7 +104,7 @@ datasource db {
 }
 ```
 
-When running a Prisma CLI command that needs the database connection URL (e.g. `prisma generate`), you need to make sure that the `DATABASE_URL` environment variable is set.
+When running a Prisma CLI command that needs the database connection URL (e.g. `prisma generate`), you need to make sure that the [DATABASE_URL](./environment-variables-reference/#database-url) environment variable is set.
```

Comment:
Note: Technically you don't have to name the env var `DATABASE_URL`
You can name it like you want like this
```
datasource db {
  provider = "postgresql"
  url      = env("MY_AWESOME_DB_ENV_VAR_URL")
}
```

Reply:
Ah good catch! Actually I wonder if this env var belongs into this reference page then because it's not an env var that you can set for the Prisma environment...

Reply:
Hmm, i guess it works by just mapping whatever to the url key. If thats the case it doesnt seem right to have this in a reference doc

Reply:
Also think it doesn't belong here as it's a user defined env var

Reply:
Cool, then i'll remove it 😄